class UrlNotFoundError(Exception):
    pass

class UrlNotAvailableError(Exception):
    pass